From f29cb69968d96105c5a89302ab15f635a77f3ba9 Mon Sep 17 00:00:00 2001
From: Dan Walkes <danwalkes@trellis-logic.com>
Date: Wed, 31 Oct 2018 14:46:43 -0600
Subject: [PATCH] Update environment defaults for tegra

Use yocto mender framework to define offset and environment related
u-boot variables.

---
 include/config_defaults.h | 17 +++++++++++------
 1 file changed, 11 insertions(+), 6 deletions(-)

Index: git/include/config_defaults.h
===================================================================
--- git.orig/include/config_defaults.h
+++ git/include/config_defaults.h
@@ -9,6 +9,8 @@
 #ifndef _CONFIG_DEFAULTS_H_
 #define _CONFIG_DEFAULTS_H_
 
+#include "config_mender_defines.h"
+
 /* Support bootm-ing different OSes */
 #define CONFIG_BOOTM_LINUX 1
 #define CONFIG_BOOTM_NETBSD 1
